cmd/get/get.go

```diff
@@ -17,6 +18,17 @@ Example get projects.
 `
 )
 
+var (
+	DefaultFetcher = NewFetcherImpl()
+)
+
+func NewFetcherImpl() interfaces.Fetcher {
+	return FetcherImpl{}
+}
+
+type FetcherImpl struct {
```

Comment:
is this only for testing? the cmdCtx being passed seems odd. Why not make this part of the command ?

Reply:
This was supposed to be used by create module when it needs to use the get functionalities.
Hence defined this interface for it.
The relaunch execution earlier requires the get for lp and exec but now we can do just directly relaunch.

But plain exec still relies on get functionalities and i was planning to add more things to this interface so that create can use this

Reply:
Will take the feedback in the followup PR.